e2e/portal/tests/TestScenarios/AdminUser/UpdateProgramRegistrationAttribute/UpdateProgramWithRegistrationAttribute.feature

```diff
@@ -0,0 +1,25 @@
+Feature: Update Program Registration Attribute
+  As an admin user
+  I want to update program registration attributes
+  So that I can modify registration field configurations and properties
+
+  Background:
```

Comment:
A lot of the features described in these feature files are covered by API tests. 

Is it intentional that we also have a feature file for these use cases?

If it is intentional feel free to resolve this. Else I can go over all feature files and list which are covered by api tests